themes/suse/_styles.scss

```diff
@@ -2,6 +2,7 @@
 
 @import "./branding.scss";
 @import "./_variables.scss";
+// @import "./_button_styles.scss";
```

Comment:
was this a reference to something in the base theme? a file that was considered for addition but abandoned?

Reply:
Doh! Good catch